packages/odyssey-react-mui/src/theme/palette.ts

```diff
@@ -16,76 +16,76 @@ import * as Tokens from "@okta/odyssey-design-tokens";
 export const palette: ThemeOptions["palette"] = {
   mode: "light",
   common: {
-    black: Tokens.ColorNeutralDark,
-    white: Tokens.ColorPaletteNeutralWhite,
+    black: Tokens.HueNeutral900,
+    white: "#ffffff",
   },
   primary: {
-    lighter: Tokens.ColorPaletteBlue000,
-    light: Tokens.ColorPaletteBlue300,
-    main: Tokens.ColorPaletteBlue500,
-    dark: Tokens.ColorPaletteBlue900,
-    contrastText: Tokens.ColorTextBodyInverse,
+    lighter: Tokens.HueBlue50,
+    light: Tokens.HueBlue300,
+    main: Tokens.HueBlue500,
+    dark: Tokens.HueBlue900,
+    contrastText: Tokens.FontColorBodyInverse,
   },
   secondary: {
     light: "#80c7ca",
-    main: Tokens.ColorPaletteTurquoise500,
+    main: "#096256",
```

Comment:
What is this?

Reply:
MUI uses them as an accent color. If we have one in the extended palette, I can include it in the tokens. That way, MUI has an on-brand fallback even if we don't utilize it.